collect_app/src/main/java/org/odk/collect/android/widgets/SelectOneSearchWidget.java

```diff
@@ -65,7 +74,6 @@ protected void createLayout() {
                 buttons.add(createRadioButton(i));
             }
         }
-
```

Comment:
I think you should not make changes in blank lines, it’s unrelated changes. A good choice is to fix these unrelated changes and ci errors in a single commit ( you can squeeze them in one commit like @lognaturel said. )
Good luck